src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,577 @@
+              "value": "COMMON"
+            }
+          ],
+          "animation_url": null
+        }
+      }
+    ]
+  },
+  "id": null
+}
+```
+</CollapsibleSection>
+
+
+## Get a list of NFT transfers {{label : 'get_nft_transfers', tag : 'API-v2'}}
+
+### Request Parameters
+
+| Parameter     | Type             | Description                                                                                               |
+| ------------- | ---------------- | ----------------------------------------------------------------------------------------------------------|
+| chains        | array            | List of chains to scan for NFTs. Chains must be [activated](/atomicdex/api/legacy/coin_activation) first. |
+| url           | string           | URL link to [Moralis API](https://docs.moralis.io/) (or proxy).                                           |
+
+
+#### 📌 Example(s)
+
+<CodeGroup title="" tag="POST" label="get_nft_transfers">
+```json {{ mm2MethodDecorate : true }}
+{
+  "userpass": "MM2_RPC_PASSWORD",
+  "method": "get_nft_transfers",
+  "mmrpc": "2.0",
+  "params": {
+    "chains": [
+      "POLYGON"
+    ],
+    "url": "https://moralis-proxy.komodo.earth"
+  }
+}
+```
+</CodeGroup>
+
+
+### Response Parameters
+
+| Parameter            | Type             | Description                                                                                      |
+| ---------------------| ---------------- | ------------------------------------------------------------------------------------------------ |
+| amount               | string           | Amount of tokens transferred.                                                                    |
+| block_hash           | string           | Hash of block in which transfer occurred.                                                        |
+| block_number         | integer          | Height of block in which transfer occurred.                                                      |
+| block_timestamp      | string           | Block date/time string. CONFIRM: SHOULD THIS NOT BE AN EPOCH TIMESTAMP?                          |
+| chain                | string           | Chain name. One of `AVALANCHE`, `BSC`, `ETH`, `FANTOM`, or `POLYGON`.                            |
+| collection_name      | string           | May be `null`. Name of collection which token belongs to.                                        |
+| contract_type        | string           | Contract type. `ERC721` or `ERC1155`.                                                            |
+| token_address        | string           | Address of token transferred.                                                                    |
+| token_id             | string           | Token ID.                                                                                        |
+| possible_spam        | boolean          | Possible spam.  MORE  INFO?                                                                      |
```

Comment:
You can provide link to [moralis feature description ](https://docs.moralis.io/web3-data-api/evm/nft-spam-detection). So there is the info

Reply:
Done.